awx/ui/src/screens/Dashboard/Dashboard.js

```diff
@@ -79,8 +79,8 @@ function Dashboard() {
         <Banner variant="info">
           <Trans>
             <p>
-              <InfoCircleIcon /> A tech preview of the new {brandName} user
-              interface can be found <a href="/ui_next">here</a>.
+              <InfoCircleIcon /> The new {brandName} user
+              interface can be found <a href="/">here</a>.
```

Comment:
I might be miss understanding this, though it reads to me as "Go to `/` to enable the new UI", which should already be the default.

Reply:
these files (`awx/ui`) are the old ui files. So if the user is navigating the old UI, the banner will point them back to new UI (`/`)